src/Application/components/Header.tsx

```diff
@@ -0,0 +1,9 @@
+import { ConnectButton } from "@rainbow-me/rainbowkit"
+
+const Header = () => (
```

Comment:
Once we settle on the UI for the various `webnative` example projects, we can probably move common components like this into their own package so they can be imported by the various projects

Reply:
Yeah for sure, I'd like to see https://github.com/fission-codes/kit with our new design. Also has React components in addition to Elm.